private/scripts/helpers.sh

```diff
@@ -2,17 +2,22 @@
 set -o pipefail
 IFS=$'\n\t'
 
-# Defines some global variables for colors.
-normal=$(tput sgr0)
-bold=$(tput bold)
-red=$(tput setaf 1)
-green=$(tput setaf 2)
-yellow=$(tput setaf 3)
+# Function to safely call tput
+safe_tput() {
+  tput "$@" 2>/dev/null || echo ""
+}
```

Comment:
is the || echo necessary if we're not using `set -e`?

Reply:
to be fair, we might add `-e` later -- I have a ticket for it, I just need to figure out if I want to just close it out or do something with it. but can remove the `||` in the meantime.

Reply:
if all its doing is preventing an error exit anyway, `|| true` should also work.